boa/src/syntax/parser/expression/lhs_expression/call_expression.rs

```diff
@@ -0,0 +1,93 @@
+use super::arguments::Arguments;
```

Comment:
Missing documentation.
```suggestion
//! Call expression parsing.
//!
//! More information:
//!  - [MDN documentation][mdn]
//!  - [ECMAScript specification][spec]
//!
//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions
//! [spec]: https://tc39.es/ecma262/#prod-CallExpression

use super::arguments::Arguments;
```